src/main/resources/db/migration/V1.0.4__init_work_types.sql

```diff
@@ -0,0 +1,18 @@
+DROP TABLE IF EXISTS work_types;
+CREATE TABLE work_types
+(
+    id TEXT,
+    label TEXT,
+    PRIMARY KEY (id)
+);
+
+INSERT INTO work_types(ID, LABEL) VALUES
```

Comment:
I know the story had something different but can we name the `id` column as `work_type_id` please?

Reply:
Let's have the identifiers as `snake_case` e.g. `hearing_work`.  This follows the DB conventions better.

Reply:
That is done